components/space-pathway-profile.jsx

```diff
@@ -0,0 +1,34 @@
+var React = require('react');
+
+var SpacePathwayProfile = React.createClass({
+  render: function() {
+    return (
+      <div className="space-pathway-profile">
+        <div className="detail">
+          <div className="header">
+            <h1>{this.props.name}</h1>
+          </div>
+          { this.props.type ? <div className="type">{this.props.type}</div> : null }
+          <div className="description" dangerouslySetInnerHTML={{__html: this.props.description}}></div>
+        </div>
+        <div className="contacts">
+          <h2>{(this.props.contacts && this.props.contacts.length > 1) ? this.props.contactTitle + "s" : this.props.contactTitle}</h2>
+          <ul>
+          {
+            this.props.contacts ? this.props.contacts.map(function(contact,i) {
+              return (
+                <li key={i}>
```

Comment:
never do this. The key is for React to do fast dom diffs, so the index in the array means nothing, the key has to be unique to the thing you're generating. So, `contact.name` for instance.